src/components/button/Button.props.ts

```diff
@@ -4,8 +4,9 @@ import { TButtonIconPosition, TButtonSize, TButtonVariant, TButtonWidth } from '
 
 export type PButton = {
     label: string;
-    disabled?: boolean;
     destructive?: boolean;
+    inverted?: boolean;
+    disabled?: boolean;
     variant?: TButtonVariant;
```

Comment:
maybe if we have the inverted, disabled and destructive props we don't need a button variant? Since the inverted will be the secondary or I am not understanding this correctly? We can add a third prop like light? or something. Either way having both separate props for styling and variant seems like we can avoid one or the other. I ran into the same dilemma with my components too lol

Reply:
not really ... all those are available in all variants ... thats the challenge here and there is likely things that change in the next few days, since the design is not 100% complete for the button atm. check the designs [here](https://www.figma.com/file/2uYWxjnMJ9IQDOba9b9bfV/Components---Buttons?node-id=1%3A184) for reference.

Reply:
I am happy to have less complex variations in the different styles though ... if you have an idea how to further simplify that i am all in on that ... right now I reduced complexity as much as i could

Reply:
We can discuss with @matthewbirtch when the designs are ready I think 👍